src/services/graphql.ts

```diff
@@ -0,0 +1,199 @@
+import gql from 'graphql-tag';
+
+// tslint:disable:max-line-length
+
+function getLineQuery() {
+    return (
+        gql`query getLineByLintunnus ($lineId: String!) {
+            linjaByLintunnus(lintunnus:$lineId) {
+                ${lineQueryFields}
+                reittisByLintunnus(first: 1, orderBy: REIVIIMPVM_DESC) {
```

Comment:
Why limit the number of reittis?

Reply:
I know we probably only use the first. But I see this as a problem since we are trying to create generic functions. This is a function for getting a line. But, this query will only return one route. What if someone starts to use this function, and don't notice that we only take the first.
Maybe we could add a comment, just for notifying why we only take the first?

Reply:
Or just modify it returning all routes

Reply:
Lets try to remove first: 1